docs/running-tests/from-local-system.md

```diff
@@ -27,38 +43,36 @@ pip install virtualenv
 ```
 
 To get the tests running, you need to set up the test domains in your
-[`hosts` file](http://en.wikipedia.org/wiki/Hosts_%28file%29%23Location_in_the_file_system).
+[`hosts` file](http://en.wikipedia.org/wiki/Hosts_%28file%29%23Location_in_the_file_system). The necessary content can be generated with `./wpt make-hosts-file`.
```

Comment:
This change seems to be repeating the information below.

Reply:
You're right! Removed.